geomstats/geometry/spd_matrices.py

```diff
@@ -23,10 +23,9 @@ def __init__(self, n):
             dim=int(n * (n + 1) / 2),
             embedding_manifold=GeneralLinear(n=n))
 
-    @staticmethod
-    def belongs(mat, atol=TOLERANCE):
+    def belongs(self, mat, atol=TOLERANCE):
         """Check if a matrix is symmetric and invertible."""
-        is_symmetric = GeneralLinear.is_symmetric(mat)
+        is_symmetric = super(SPDMatrices, self).belongs(mat, atol)
```

Comment:
Why this change?

Reply:
to check the size of the matrix corresponds to the `n` the object was instantiated with